plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryMetadata.java

```diff
@@ -468,6 +493,24 @@ public Optional<ConstraintApplicationResult<ConnectorTableHandle>> applyFilter(
         return Optional.of(new ConstraintApplicationResult<>(updatedHandle, constraint.getSummary(), false));
     }
 
+    @Override
+    public Optional<TableFunctionApplicationResult<ConnectorTableHandle>> applyTableFunction(ConnectorSession session, ConnectorTableFunctionHandle handle)
```

Comment:
@martint @kasiafi Could you please review around `applyTableFunction()` in this PR when you have time?

Reply:
@martint @kasiafi Gentle reminder.